pkgs/development/python-modules/arxiv/default.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  feedparser,
+  requests,
+
+  # tests
+  mock,
+  pytestCheckHook,
+}:
+buildPythonPackage rec {
+  pname = "arxiv";
+  version = "2.1.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "lukasschwab";
+    repo = "arxiv.py";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Niu3N0QTVxucboQx1FQq1757Hjj1VVWeDZn7O7YtjWY=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    feedparser
+    requests
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    mock
+  ];
+
+  disabledTests = [
+    "test_from_feed_entry"
```

```suggestion
  disabledTests = [
    # Require network access
    "test_from_feed_entry"
```
Please leave a comment on why disable these tests.